internal/quickstart/show_sdk_instructions.go

```diff
@@ -71,12 +83,21 @@ func (m showSDKInstructionsModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		case key.Matches(msg, pressableKeys.Enter):
 			// TODO: only if all data are fetched?
 			cmd = sendShowToggleFlagMsg()
+		default:
+			m.viewport, cmd = m.viewport.Update(msg)
 		}
+	case tea.MouseMsg:
+		m.viewport, cmd = m.viewport.Update(msg)
 	case fetchedSDKInstructions:
 		m.instructions = sdks.ReplaceFlagKey(string(msg.instructions), m.flagKey)
 	case fetchedEnv:
 		m.sdkKey = msg.sdkKey
 		m.instructions = sdks.ReplaceSDKKey(string(m.instructions), msg.sdkKey)
+		md, err := m.renderMarkdown()
+		if err != nil {
+			return m, sendErr(err)
+		}
+		m.viewport.SetContent(md)
```

Comment:
rendering once here vs. every time the `View` method is called, to allow for proper updating